libfwupdplugin/fu-lzma-common.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#ifdef HAVE_LZMA
+#include <lzma.h>
+#endif
+
+#include "fu-lzma-common.h"
+
+/**
+ * fu_lzma_decompress_bytes:
+ * @blob: data
+ * @error: (nullable): optional return location for an error
+ *
+ * Decompresses a LZMA stream.
+ *
+ * Returns: decompressed data
+ *
+ * Since: 1.6.2
```

Comment:
Needs bump

Reply:
Yup, needs to be in contrib/migrate.py too -- fixed.